test/Swashbuckle.AspNetCore.IntegrationTests/SwaggerVerifyIntegrationTest.cs

```diff
@@ -111,6 +114,25 @@ private static async Task<string> SwaggerResponse(HttpClient client, string swag
             return contentStream;
         }
 
+        /// <summary>
+        /// Normalize '\n' strings into '\r\n' which is expected linebreak in Verify verified.txt files.
+        /// </summary>
+        private static StringBuilder NormalizeLineBreaks(string swagger)
+        {
+            var output = new StringBuilder(swagger.Length);
+            for (var i = 0; i < swagger.Length; i++)
+            {
+                if (swagger[i] == '\\' && i > 0 && i < swagger.Length - 1 && swagger[i+1] == 'n' && swagger[i - 1] != 'r')
+                {
+                    output.Append("\\r");
+                }
+
+                output.Append(swagger[i]);
+            }
+
+            return output;
+        }
```

Comment:
Maybe a better solution to this would be to read `swagger` into a `StringReader`, then iterate through it line-by-line (so the framework does the parsing for us), and then emit each line read into `output`.

Something like:

```csharp
using var reader = new StringReader(swagger);

string line;

while ((line = reader.ReadLine()) is not null)
{
    output.Append(line);
    output.Append("\r\n");
}
```

Reply:
it doesn't help much because these `\n` or `\r\n` are part of description field in json and they are actually string values, not new lines characters, so they are not parsed by framework (or string.ReplaceLineEndings)

I managed to simplify method by using regex